x/stakeibc/keeper/icqcallbacks_withdrawal_reward_balance.go

```diff
@@ -59,33 +55,14 @@ func WithdrawalRewardBalanceCallback(k Keeper, ctx sdk.Context, args []byte, que
 	k.Logger(ctx).Info(utils.LogICQCallbackWithHostZone(chainId, ICQCallbackID_WithdrawalRewardBalance,
 		"Query response - Withdrawal Reward Balance: %v %s", withdrawalRewardBalanceAmount, tradeRoute.RewardDenomOnHostZone))
 
-	// Split the withdrawal amount into a rebate, stride fee, and reinvest portion
-	rebateAmount, tradeAmount, err := k.CalculateRewardsSplitBeforeRebate(ctx, hostZone, withdrawalRewardBalanceAmount)
-	if err != nil {
-		return errorsmod.Wrapf(err, "unable to check for rebate amount")
-	}
-
-	// If there's a rebate portion, fund the community pool with that amount
-	if rebateAmount.GT(sdkmath.ZeroInt()) {
-		rebateToken := sdk.NewCoin(tradeRoute.RewardDenomOnHostZone, rebateAmount)
-		if err := k.FundCommunityPool(ctx, hostZone, rebateToken, types.ICAAccountType_WITHDRAWAL); err != nil {
-			return errorsmod.Wrapf(err, "unable to submit fund community pool ICA")
-		}
-
-		k.Logger(ctx).Info(utils.LogICQCallbackWithHostZone(chainId, ICQCallbackID_WithdrawalRewardBalance,
-			"Sending rebate tokens %v %s to community pool",
-			rebateAmount, tradeRoute.RewardDenomOnRewardZone))
-	}
-
-	// Transfer the amount leftover after to the rebate to the trade zone so it can be swapped for the native token
-	// We transfer both the amount to be reinvested, and the amount for the stride fee
-	if err := k.TransferRewardTokensHostToTrade(ctx, tradeAmount, tradeRoute); err != nil {
+	// Transfer the reward amount to the trade zone so it can be swapped for the native token
```

Comment:
Should we check for 0 amount? I see `UnmarshalAmountFromBalanceQuery` handles the case where the acct is uninitialized, but not clear it handles the 0 balance case - we prob don't want to send a `TransferRewardTokensHostToTrade` if the bal is 0.

Reply:
`TransferRewardTokensHostToTrade` doesn't send a transfer if the amount is less than the min configured amount